plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/complete-onboarding-wizard.spec.js

```diff
@@ -268,7 +268,8 @@ test.describe(
 	}
 );
 
-test.describe( 'Store owner can go through setup Task List', () => {
+// Skipping this test because it's very flaky.
+test.describe.skip( 'Store owner can go through setup Task List', () => {
```

Comment:
Skipping this test means that the store currency doesn't get set to USD which is a (unintended) pre-condition for subsequent tests.